src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -197,6 +197,8 @@ default boolean onBlockStartBreak(ItemStack itemstack, BlockPos pos, Player play
      * @param count  The amount of time in tick the item has been used for
      *               continuously
      */
+    /** @deprecated use {@link Item#onUseTick(Level,LivingEntity,ItemStack,int)} */
+    @Deprecated(since = "1.20")
```

Comment:
This new javadoc should be merged (appended) into the existing javadoc. The annotation should be marked as `since = "1.19.3"` and `forRemoval = true`.

Additionally, it should probably be noted that a difference between this and the method in `Item` is that the `Item` method can be called with a `useItemRemaining` of `-1`, if the corresponding `LivingEntityUseItemEvent.Tick` is cancelled or modified in that way.